routes/index.js

```diff
@@ -4,7 +4,11 @@ const express = require('express')
 const router = express.Router()
 
 router.get('/', function(req, res) {
-  res.status(200).send({ status: 'OK' })
+  const sha = require('child_process')
+    .execSync('git rev-parse HEAD')
+    .toString().trim()
+  const msg = `{ "status": "OK", "sha": "${sha}" }`
```

Comment:
execSync can throw, so probably need to wrap in a try-catch block

this endpoint can potentially be the k8s health check. In Azure app service, there is also a similar concept. In the service log, this seemed to be called every 5 minutes (likely as health check). Retrieving the commit hash every time may not be performant.  Is it possible to cache this hash or store this commit hash in an environment variable?

Reply:
Updated the deploy to pass the sha as a config.  It now gets the sha once during setup from the configs.